src/metrics/utils.jl

```diff
@@ -70,17 +80,40 @@ Convenience method that slices the data in the specified manner.
 - `metric` : Function, the metric function to apply to "raw" data.
 - `data` : YAXArray, data to pass into `metric`
 - `args` : Additional positional arguments to pass into `metric`
-- `dims` : dummy keyword argument, not used but defined to allow use with other methods
+- `kwargs` : Additional keyword arguments to pass into `slice_results`
+    - `dims` : dummy keyword argument, not used but defined to allow use with other methods
 """
 function call_metric(metric::Union{Function,Metric}, data::YAXArray, args...; kwargs...)
-    dims = haskey(kwargs, :dims) ? kwargs[:dims] : nothing
+    dims = get(kwargs, :dims, nothing)
     if isnothing(dims)
         return metric(slice_results(data; kwargs...), args...)
     else
         return metric(slice_results(data; kwargs...), args...; dims=dims)
     end
 end
 
+"""
+    fill_axes_properties(datacube::YAXArray)::YAXArray
+
+Fill :axes_names and :axes_units properties of datacube
+
+# Arguments
+- `datacube` : YAXArray datacube
+"""
+function fill_axes_properties(metric::Metric, metric_result::YAXArray)::YAXArray
```

Comment:
Argument list in docstring does not match implementation?

Reply:
Done